### PR TITLE
refactored authentication to use the ui-shared

### PR DESCRIPTION
**Pull request by edewit**
Signed-off-by: Erik Jan de Wit <erikjan.dewit@gmail.com>

### CODE REVIEW

**Review by ssilvert — DISMISSED** [review]
LGTM

**Review by jonkoops — DISMISSED** [review]
This cleans up really nice! LGTM

File: convert.java
Comment (jonkoops) [inline convert.java:1]: This seems to have been checked in, but I don't think it was supposed to.

**Review by jonkoops — CHANGES_REQUESTED** [review]

File: js/apps/admin-ui/maven-resources/theme/keycloak.v2/admin/messages/messages_en.properties.orig
Comment (jonkoops) [inline js/apps/admin-ui/maven-resources/theme/keycloak.v2/admin/messages/messages_en.properties.orig:1]: Looks like this file was checked in unintentionally.

**Review by jonkoops — CHANGES_REQUESTED** [review]

File: js/apps/admin-ui/src/App.tsx.orig [hunk 1]
```diff
@@ -0,0 +1,93 @@
...
+          </RecentRealmsProvider>
+        </WhoAmIContextProvider>
+      </RealmContextProvider>
+    </RealmsProvider>
+  </ErrorBoundaryProvider>
+);
+
+export const App = () => {
+  return (
+<<<<<<< HEAD
```
Comment (jonkoops) [inline js/apps/admin-ui/src/App.tsx.orig:47]: Merge conflict here.

**Review by jonkoops — CHANGES_REQUESTED** [review]

**Review by jonkoops — APPROVED** [review]

**Pull request merged**